src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,15 +1,33 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private double diggingDepth;
+    private double bucketVolume;
+
     public Excavator() {
     }
 
+    public double getDiggingDepth() {
+
+        return diggingDepth;
+    }
+
+    public void setDiggingDepth(double diggingDepth) {
+
+        this.diggingDepth = diggingDepth;
+    }
+
+    public double getBucketVolume() {
+        return bucketVolume;
+    }
+
+    public void setBucketVolume(double bucketVolume) {
+        this.bucketVolume = bucketVolume;
+    }
+
     @Override
     public void doWork() {
+
         System.out.println("Excavator started to work");
```

Comment:
![image](https://github.com/mate-academy/jv-pecs/assets/20326858/36efce9c-f1d5-4d1d-835b-fe6d309d3572)

Reply:
Sorry, I'll fix it now. I just checked it in the program and everything went well! Didn't finish watching(((